src/static/css/module/prepare-worksheets.less

```diff
@@ -0,0 +1,112 @@
+/* ==========================================================================
+   "Home buyer's preparation worksheets" custom styles
+   ========================================================================== */
+
+/* Preparation worksheet specific styles */
+
+// Width of right-hand worksheet column (priority, likely-ness, etc).
+@right-column-width: 202px;
+
+// Colors
+@btn-grade-disabled-color: @gray-80;
+@btn-grade-unselected-color: @gray-50;
+@btn-grade-selected-color: @darkgray;
+@btn-grade-hover-color: @gray;
+
+@btn-grade-disabled-text-color: @white;
+@btn-grade-unselected-text-color: @darkgray;
+@btn-grade-selected-text-color: @white;
+@btn-grade-hover-text-color: @white;
+
+@btn-delete-color: @gray;
+@btn-delete-hover-color: @gray-20;
+
+// Inputs with buttons for goals, risks, and red flags.
+.input-with-btns_btns { 
+  float: right;
+}
+
+.input-with-btns_input { 
+  display: block;
+  overflow: hidden;
+  padding-right: 10px;
+}
+
+.input-with-btns_input input { 
+  width: 100%;
+}
+
+.input-with-btns {
+  display: block;
+  clear: both;
+  margin-bottom: 10px;
+}
+
+// Preparation worksheets goals, risks, red flag icon colors.
+.prepare-worksheet .cf-icon-approved-round {
+  color: @green;
+}
+
+.prepare-worksheet .cf-icon-complaint-round {
+  color: @redorange;
+}
+
+.prepare-worksheet .cf-icon-error-round {
+  color: @gold;
+}
+
+// Inputs with buttons for goals, risks, and red flags.
+// The color settings override CF colors.
+
+// Right-hand column of the worksheets (priority, likely-ness, etc).
+.worksheet .u-right {
```

Comment:
**Suggestion:** It would be nice if this was more declarative. `.worksheets-header-col` or something. This also has the benefit of allowing you to add `.u-right` in the CSS instead of the HTML:

```less
.worksheets-header-col {
  .u-right();
  width: @right-column-width;
}
```

```html
<h4 class="worksheets-header-col">Priority level</h4>
```